pages/index.jsx

```diff
@@ -8,10 +8,14 @@ import Pontoon from '../components/pontoon.jsx';
 var Index = React.createClass({
   render: function() {
     var metaData = this.props.metaData;
+    var robots = 'index, follow';
     var googleFonts = "https://fonts.googleapis.com/css?family=Open+Sans:600,400,300,300italic";
     if (this.props.localeInfo === "cs") {
       googleFonts += "&subset=latin-ext";
     }
+    if (metaData.current_url.indexOf('thank-you') !== -1) {
```

Comment:
That's just MDN being lazy. You can also do.
```
'Blue Whale'.indexOf('Blue') >= 0; // true
'Blue Whale'.indexOf('Bloe') >= 0; // false
'Blue Whale'.indexOf('Blue') > -1; // true
'Blue Whale'.indexOf('Bloe') > -1; // false
```
These have the advantages of not being double negatives.

If you were to tell someone this logic in English, would you say "'Blue' is not not in 'Blue Whale'" ? or would you just say "'Blue' is in 'Blue Whale'".

Did MDN give a reason? `> -1` might be the fastest too http://jsperf.com/indexof-test-test-test-test

I think it's ok to question MDN :P